.gitignore

```diff
@@ -24,4 +24,5 @@ yarn-error.log*
 
 /.vscode
 
-.eslintcache
\ No newline at end of file
+.eslintcache
+debug.log
```

Comment:
Github  prefers that every file ends with a blank line or new line character.  It isn't necessary but its a good practice to get into, so add one  here